### PR TITLE
Update the numbered list to match the graphic

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/security/your-apis/openid-connect.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ Tyk comes with support for OpenID Connect Identity Tokens provided by any standa
...
-5.  Tyk Gateway validates OIDC ID Token signature.
-6.  Tyk Gateway checks the IDP is a recognised IDP (registered as approved).
-7.  Tyk verifies the client ID as one that is trusted and pre-registered with Tyk Gateway.
+1.  A User logs in via a supported OIDC Provider to request access to their resource.
+2.  The User gains access to the Provider and uses their service.
+3.  The Identity Provider generates OAuth token set and OIDC ID Token The ID Token is signed by provider with their public key. This is then returned to the user.
+4.  The User's client utilises OIDC ID Token as access token for an API managed by Tyk Gateway.
+5.  The Tyk Gateway checks the IDP is an approved IDP (registered as approved). The Gateway validates the OIDC ID Token signature.
+6.  The Tyk Gateway verifies the client ID as one that is trusted and pre-registered with Tyk Gateway.
+7.  The Gateway Checks the OIDC token signature against Auth0's public key set.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/openid-connect.md]: check not Check?

File: tyk-docs/content/security/your-apis/openid-connect.md [hunk 2]
```diff
@@ -11,15 +11,16 @@ Tyk comes with support for OpenID Connect Identity Tokens provided by any standa
...
-1.  User requests access to resource via a supported OIDC Provider (e.g. Google).
-2.  User logs into resource provider and grants scope access to their data.
-3.  Identity Provider generates OAuth token set and OIDC ID Token, ID Token is signed by provider with their public key.
-4.  User's client utilises OIDC ID Token as access token for an API managed by Tyk Gateway.
-5.  Tyk Gateway validates OIDC ID Token signature.
-6.  Tyk Gateway checks the IDP is a recognised IDP (registered as approved).
-7.  Tyk verifies the client ID as one that is trusted and pre-registered with Tyk Gateway.
+1.  A User logs in via a supported OIDC Provider to request access to their resource.
+2.  The User gains access to the Provider and uses their service.
+3.  The Identity Provider generates OAuth token set and OIDC ID Token The ID Token is signed by provider with their public key. This is then returned to the user.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/openid-connect.md]: Where's the comma?

Comment (letzya) [inline tyk-docs/content/security/your-apis/openid-connect.md]: Where's the comma? also doesn't read well.

Maybe:
The Identity Provider generates OIDC  `id_token` which is signed by provider with their public key and return it to the user

File: tyk-docs/content/security/your-apis/openid-connect.md [hunk 3]
```diff
@@ -11,15 +11,16 @@ Tyk comes with support for OpenID Connect Identity Tokens provided by any standa
...
 ### The OIDC Flow:
 
-1.  User requests access to resource via a supported OIDC Provider (e.g. Google).
-2.  User logs into resource provider and grants scope access to their data.
-3.  Identity Provider generates OAuth token set and OIDC ID Token, ID Token is signed by provider with their public key.
-4.  User's client utilises OIDC ID Token as access token for an API managed by Tyk Gateway.
-5.  Tyk Gateway validates OIDC ID Token signature.
-6.  Tyk Gateway checks the IDP is a recognised IDP (registered as approved).
-7.  Tyk verifies the client ID as one that is trusted and pre-registered with Tyk Gateway.
+1.  A User logs in via a supported OIDC Provider to request access to their resource.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/openid-connect.md]: The original line is preferred

File: tyk-docs/content/security/your-apis/openid-connect.md [hunk 4]
```diff
@@ -11,15 +11,16 @@ Tyk comes with support for OpenID Connect Identity Tokens provided by any standa
...
+2.  The User gains access to the Provider and uses their service.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/openid-connect.md]: The original line is preferred

File: tyk-docs/content/security/your-apis/openid-connect.md [hunk 5]
```diff
@@ -11,15 +11,16 @@ Tyk comes with support for OpenID Connect Identity Tokens provided by any standa
...
+5.  The Tyk Gateway checks the IDP is an approved IDP (registered as approved). The Gateway validates the OIDC ID Token signature.
+6.  The Tyk Gateway verifies the client ID as one that is trusted and pre-registered with Tyk Gateway.
+7.  The Gateway Checks the OIDC token signature against Auth0's public key set.
 8.  If the client ID is valid, Tyk applies the policy ID matched with this client to the user session.
-9.  Tyk then validates the users session according to the quotas, rate limits and access rules for the matching policy for *either* the bearer of the token across all clients they use from this IDP *or* validates the session on a per client / per identity basis, e.g. User Alice will have different Access Rules depending on whether they are using a mobile client or a web client.
+9.  Tyk then validates the users session according to the quotas, rate limits and access rules for the matching policy for **either** the bearer of the token across all clients they use from this IDP **or** validates the session on a per client / per identity basis. For example, user Alice will have different Access Rules depending on whether they are using a mobile client or a web client.
```
Comment (letzya) [inline tyk-docs/content/security/your-apis/openid-connect.md]: this change is fine

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/security/your-apis/openid-connect.md (hunk 3 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/openid-connect.md]: Change made

File: tyk-docs/content/security/your-apis/openid-connect.md (hunk 4 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/openid-connect.md]: Done

File: tyk-docs/content/security/your-apis/openid-connect.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/openid-connect.md]: Done

File: tyk-docs/content/security/your-apis/openid-connect.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/security/your-apis/openid-connect.md]: Changed

**Pull request merged**